addons/account/wizard/account_move_send_views.xml

```diff
@@ -15,16 +15,20 @@
                 <field name="enable_download" invisible="1"/>
                 <field name="enable_send_mail" invisible="1"/>
                 <field name="send_mail_readonly" invisible="1"/>
+                <field name="send_mail_warning_message" invisible="1"/>
                 <field name="display_mail_composer" invisible="1"/>
                 <field name="mail_lang" invisible="1"/>
-                <field name="mail_partner_ids" widget="many2many_tags_email" class="d-none" context="{'force_email': True}"/>
 
                 <!-- Warnings -->
                 <div name="warnings">
-                    <div class="alert alert-warning"
-                        role="alert"
-                        invisible="not send_mail_warning_message">
-                        <field name="send_mail_warning_message"/>
+                    <span invisible="send_mail_warning_message != 'missing_info'"><b>E-mail</b></span>
```

Comment:
We should probably check with a PO if we want to have those "warning titles" :slightly_smiling_face:

Reply:
Edit: I checked with CHKL and he confimed the "warning title/warning sections" is too much for the UI, let's just remove that part :slightly_smiling_face:

Reply:
In Italy we have all sorta checks on the invoices themselves, the partner, its address etc. etc.
They are not necessary for sending the invoice via email. There might be confusion, imho. I felt it myself.